blogg/posts/views.py

```diff
@@ -0,0 +1,10 @@
+from django.http import JsonResponse
+from .posts import get_slugs, load_post
+
+
+def post(request, slug):
+    return JsonResponse(load_post(slug))
```

Comment:
All we need here is a `pass` command.
This is designed for the candidate to download and complete - this is a bit too much help for them.